src/javascript/app/base/header.js

```diff
@@ -547,21 +561,74 @@ const Header = (() => {
         Client.sendLogoutRequest();
     };
 
+    const bindHeaders = () => {
+        const high_risk_accounts_accordion_header = document.getElementById('high_risk_accounts');
+        const low_risk_non_eu_accordion_header = document.getElementById('low_risk_accounts_non_eu');
+        const low_risk_eu_accordion_header = document.getElementById('low_risk_accounts_eu');
+        const low_risk_eu_container = document.getElementById('account__switcher-accordion-eu');
+        if (Client.isHighRisk() || isEuCountry()) {
+            high_risk_accounts_accordion_header.style.display = 'flex';
+            low_risk_non_eu_accordion_header.style.display    = 'none';
+            low_risk_eu_accordion_header.style.display        = 'none';
+            low_risk_eu_container.style.display               = 'none';
+        } else if (Client.isLowRisk()) {
+            high_risk_accounts_accordion_header.style.display = 'none';
+            low_risk_non_eu_accordion_header.style.display    = 'flex';
+            low_risk_eu_accordion_header.style.display        = 'flex';
+            $('<div class="account__switcher-seperator" />').insertBefore('#account__switcher-accordion-eu');
+        }
+    };
+
+    const mapCurrencyName = (currency) => {
+        let currencyName;
+        if (currency === 'USD') {
+            currencyName = 'US Dollar';
+        } else if (currency === 'EUR') {
+            currencyName = 'Euro';
+        } else if (currency === 'GBP') {
+            currencyName = 'Pound Sterling';
+        } else if (currency === 'AUD'){
+            currencyName = 'Australian Dollar';
+        } else if (currency === 'BTC'){
+            currencyName = 'Bitcoin';
+        } else if (currency === 'ETH') {
+            currencyName = 'Ethereum';
+        } else if (currency === 'USDT') {
+            currencyName = 'Tether Omni';
+        } else if (currency === 'eUSDT') {
+            currencyName = 'Tether ERC20';
+        } else if (currency === 'USDC') {
+            currencyName = 'USD Coin';
+        } else if (currency === 'LTC'){
+            currencyName = 'Litecoin';
+        } else {
+            currencyName = currency;
+        }
+        return currencyName;
+    };
```

Comment:
@henry-deriv let's try extending and using getCurrencies() from '../../_common/base/currency_base', or if this list of currencies should not include any other currencies, please add a constant for these currencies in '../../_common/utility':
```
    const displayed_currencies = {
        USD: localize('US Dollar'),
        EUR: localize('Euro'),
        GBP: localize('Pound Sterling'),
        AUD: localize('Australian Dollar'),
        BTC: localize('Bitcoin'),
        ETH: localize('Ethereum'),
        USDT: localize('Tether Omni'),
        eUSDT: localize('Tether ERC20'),
        USDC: localize('USD Coin'),
        LTC: localize('Litecoin'),
    };
```

```suggestion
```
 and please import it like this in this file e.g.:
`const displayed_currencies = require('../../_common/utility').displayed_currencies;`
and please use it instead of `mapCurrencyName(currency)` at line 623🙂